{
    "type": [
        r"""TypeError: '>=' not supported between instances of 'numpy.ndarray' and 'Timestamp'""",
        r"""TypeError: '>=' not supported between instances of 'str' and 'datetime.date'""",
        r"""TypeError: 'Collection' object is not iterable""",
        r"""TypeError: 'int' object is not subscriptable""",
        r"""TypeError: 'months' is an invalid keyword argument for __new__()""",
        r"""TypeError: 'months' is an invalid keyword argument for __new__()""",
        r"""TypeError: __init__() got an unexpected keyword argument 'hostname'""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: argument of type 'method' is not iterable""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: eval() arg 1 must be a string, bytes or code object""",
        r"""TypeError: initial_value must be str or None, not DataFrame""",
        r"""TypeError: initial_value must be str or None, not DataFrame""",
        r"""TypeError: not enough arguments for format string""",
        r"""TypeError: strptime() argument 1 must be str, not datetime.date""",
        r"""TypeError: strptime() argument 1 must be str, not datetime.date""",
        r"""TypeError: strptime() argument 1 must be str, not datetime.date""",
        r"""TypeError: the JSON object must be str, bytes or bytearray, not DataFrame""",
        r"""TypeError: unsupported operand type(s) for *: 'decimal.Decimal' and 'float'""",
        r"""TypeError: unsupported operand type(s) for *: 'decimal.Decimal' and 'float'""",
        r"""TypeError: unsupported operand type(s) for *: 'decimal.Decimal' and 'float'""",
        r"""TypeError: unsupported operand type(s) for *: 'decimal.Decimal' and 'float'""",
        r"""TypeError: unsupported operand type(s) for *: 'float' and 'decimal.Decimal'""",
        r"""TypeError: unsupported operand type(s) for *: 'float' and 'decimal.Decimal'""",
        r"""TypeError: unsupported operand type(s) for +=: 'float' and 'decimal.Decimal'""",
        r"""TypeError: unsupported operand type(s) for +=: 'float' and 'decimal.Decimal'""",
        r"""TypeError: unsupported operand type(s) for +=: 'float' and 'decimal.Decimal'""",
        r"""TypeError: unsupported operand type(s) for /: 'decimal.Decimal' and 'float'""",
        r"""ValueError: Invalid file path or buffer object type: <class 'NoneType'>""",
        r"""ValueError: Invalid file path or buffer object type: <class 'pandas.core.frame.DataFrame'>""",
        r"""ValueError: Invalid file path or buffer object type: <class 'pandas.core.frame.DataFrame'>""",
        r"""ValueError: Invalid file path or buffer object type: <class 'pandas.core.frame.DataFrame'>""",
        r"""ValueError: Invalid file path or buffer object type: <class 'pandas.core.frame.DataFrame'>""",
        r"""ValueError: Invalid file path or buffer object type: <class 'pandas.core.frame.DataFrame'>""",
        r"""ValueError: Invalid file path or buffer object type: <class 'pandas.core.frame.DataFrame'>""",
        r"""ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().""",
        r"""ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().""",
        r"""ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().""",
        r"""ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().""",
        r"""ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().""",
        r"""ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().""",
        r"""ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().""",
        r"""ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().""",
    ],
    "library": [
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: 'DataFrame' object has no attribute 'decode'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_dataframe'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas' has no attribute 'read_msgpack'""",
        r"""AttributeError: module 'pandas.compat' has no attribute 'StringIO'""",
        r"""AttributeError: module 'pymongo' has no attribute 'regex'""",
bson.errors.InvalidDocument: cannot encode object: Decimal('5021.995675799087'), of type: <class 'decimal.Decimal'>
        r"""ImportError: cannot import name 'json_normalize' from 'pandas.io.json' (/usr/local/lib/python3.8/dist-packages/pandas/io/json/__init__.py)""",
        r"""ImportError: cannot import name 'json_normalize' from 'pandas.io.json' (/usr/local/lib/python3.8/dist-packages/pandas/io/json/__init__.py)""",
        r"""ModuleNotFoundError: No module named 'redis_functions'""",
        r"""pymongo.errors.ConfigurationError: Unknown option database""",
        r"""pymongo.errors.ConfigurationError: Unknown option database""",
        r"""pymongo.errors.ConfigurationError: Unknown option db""",
        r"""pymongo.errors.ConfigurationError: Unknown option db""",
        r"""pymongo.errors.ConfigurationError: Unknown option hostname""",
pymongo.errors.InvalidOperation: Cannot use MongoClient after close
        r"""pymongo.errors.OperationFailure: Cannot do exclusion on field P_NAME in inclusion projection, full error: {'ok': 0.0, 'errmsg': 'Cannot do exclusion on field P_NAME in inclusion projection', 'code': 31254, 'codeName': 'Location31254'}""",
        r"""pymysql.err.OperationalError: (1305, 'FUNCTION tpch.strftime does not exist')""",
        r"""pymysql.err.OperationalError: (1305, 'FUNCTION tpch.strftime does not exist')""",
        r"""ValueError: 6 columns passed, passed data had 7 columns""",
        r"""ValueError: DataFrame constructor not properly called!""",
        r"""ValueError: If using all scalar values, you must pass an index""",
        r"""ValueError: max() arg is an empty sequence""",
        r"""ValueError: No objects to concatenate""",
    ],
    "schema": [
        r"""': (1054, "Unknown column 'C_CUSTKEY' in 'field list'")""",
        r"""': (1054, "Unknown column 'l_extendedprice' in 'field list'")""",
        r"""': (1146, "Table 'tpch.nation' doesn't exist")""",
        r"""': (1146, "Table 'tpch.orders' doesn't exist")""",
        r"""': (1146, "Table 'tpch.supplier' doesn't exist")""",
        r"""': (1146, "Table 'tpch.supplier' doesn't exist")""",
        r"""KeyError: "Only a column name can be used for the key in a dtype mappings argument. 'N_NATIONKEY' not found in columns."""",
        r"""KeyError: 'C_CUSTKEY'""",
        r"""KeyError: 'C_CUSTKEY'""",
        r"""KeyError: 'L_EXTENDEDPRICE'""",
        r"""KeyError: 'L_EXTENDEDPRICE'""",
        r"""KeyError: 'L_EXTENDEDPRICE'""",
        r"""KeyError: 'L_ORDERKEY'""",
        r"""KeyError: 'L_ORDERKEY'""",
        r"""KeyError: 'L_ORDERKEY'""",
        r"""KeyError: 'L_PARTKEY'""",
        r"""KeyError: 'L_PARTKEY'""",
        r"""KeyError: 'L_PARTKEY'""",
        r"""KeyError: 'L_PARTKEY'""",
        r"""KeyError: 'L_SHIPMODE'""",
        r"""KeyError: 'N_REGIONKEY'""",
        r"""KeyError: 'O_CUSTKEY'""",
        r"""KeyError: 'O_ORDERDATE'""",
        r"""KeyError: 'P_BRAND'""",
        r"""KeyError: 'P_PARTKEY'""",
        r"""KeyError: 'PS_SUPPKEY'""",
        r"""KeyError: 'SAUDI ARABIA'""",
        r"""NameError: name 'N_NATIONKEY' is not defined""",
        r"""NameError: name 'supplier_data' is not defined""",
        r"""NameError: name 'tpch_mongodb_conn_info' is not defined""",
        r"""pandas.errors.DatabaseError: Execution failed on sql 'SELECT * FROM orders': (1146, "Table 'tpch.orders' doesn't exist")""",
        r"""pandas.errors.DatabaseError: Execution failed on sql 'SELECT * FROM supplier WHERE S_NATIONKEY IN (SELECT N_NATIONKEY FROM nation WHERE N_NAME = 'GERMANY')': (1146, "Table 'tpch.nation' doesn't exist")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1049, "Unknown database 'tpch'")""",
        r"""pymysql.err.OperationalError: (1054, "Unknown column 'C_CUSTKEY' in 'where clause'")""",
        r"""pymysql.err.OperationalError: (1054, "Unknown column 'N_NATIONKEY' in 'field list'")""",
        r"""pymysql.err.OperationalError: (1054, "Unknown column 'S_NATIONKEY' in 'field list'")""",
        r"""pymysql.err.OperationalError: (1054, "Unknown column 'S_NATIONKEY' in 'where clause'")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.customer' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.customer' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.customer' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.lineitem' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.lineitem' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.lineitem' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.nation' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.nation' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.nation' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.nation' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.nation' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.nation' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.orders' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.orders' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.orders' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.orders' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.orders' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.part' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.partsupp' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.partsupp' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.partsupp' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.partsupp' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.supplier' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.supplier' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.supplier' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.supplier' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.supplier' doesn't exist")""",
        r"""pymysql.err.ProgrammingError: (1146, "Table 'tpch.supplier' doesn't exist")""",
    ],
    "runtime": [
        r"""IndexError: list index out of range""",
        r"""pandas.errors.MergeError: Passing 'suffixes' which cause duplicate columns {'_id_x'} is not allowed.""",
        r"""UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 0: invalid start byte""",
        r"""UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 0: invalid start byte""",
        r"""UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 0: invalid start byte""",
        r"""UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 0: invalid start byte""",
    ],
    "syntax": [
        r"""pymysql.err.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ') THEN 1\n            ELSE 0\n    END) AS HIGH_LINE_COUNT,\n    SUM(CASE\n          ' at line 4")""",
        r"""pymysql.err.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')\nGROUP BY O_ORDERPRIORITY\nORDER BY O_ORDERPRIORITY' at line 4")""",
        r"""pymysql.err.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '53 AND L_QUANTITY < 4.0333333' at line 3")""",
        r"""pymysql.err.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '53 AND L_QUANTITY < 4.0333333' at line 3")""",
        r"""SyntaxError: invalid syntax""",
    ],
    "connection": [
        r"""redis.exceptions.ConnectionError: Error 99 connecting to localhost:6379. Cannot assign requested address.""",
        r"""pymysql.err.Error: Already closed""",
        r"""pymysql.err.Error: Already closed""",
        r"""pymysql.err.Error: Already closed""",
        r"""UnboundLocalError: local variable 'connection' referenced before assignment""",
    ],
}
